README.rdoc

```diff
@@ -0,0 +1,21 @@
+== README
```

Comment:
We'll have to rework the readme as it's open sourced. Also we usually use markdown rather than rdoc